bundles/org.openhab.ui/web/src/components/cards/glance/location/status-badge.vue

```diff
@@ -72,16 +72,16 @@ export default {
       let direct, equipment, allPoints, points
       switch (this.type) {
         case 'battery':
-          direct = findPoints(this.element.properties, 'Point_Status_LowBattery', true)
+          direct = findPoints(this.element.properties, 'Point', true, 'Property_LowBattery')
           if (direct.length) return direct
-          return findPoints(allEquipmentPoints(this.element.equipment), 'Point_Status_LowBattery', true)
+          return findPoints(allEquipmentPoints(this.element.equipment), 'Point', true, 'Property_LowBattery')
         case 'lights':
-          return [...this.queryLightPoints, ...this.queryLightEquipment('Equipment_Lightbulb'), ...this.queryLightEquipment('Equipment_LightStripe')]
+          return [...this.queryLightPoints, ...this.queryLightEquipment('Equipment_LightSource')]
```

Comment:
Does `Equipment_LightSource` include all different light equipments, i.e. if I tag with `LightBulb`, will core automatically add `Equipment_LightSource` as well?

Reply:
Lightbulb is a child of LightSource so there should be a badge.

.. but perhaps you can check and confirm it?

Reply:
I just checked and it seems this will work as expected 👍